modules/test/playwright/tests/object-web/listTypeDefinitions.spec.ts

```diff
@@ -0,0 +1,90 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2024 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import {expect, mergeTests} from '@playwright/test';
+
+import {apiHelpersTest} from '../../fixtures/apiHelpersTest';
+import {listTypeDefinitionsPagesTest} from '../../fixtures/listTypeDefinitionsPagesTest';
+import {loginTest} from '../../fixtures/loginTest';
+import {siteSettingsPageTests} from '../../fixtures/siteSettingsPagesTest';
+import {getRandomInt} from '../../utils/getRandomInt';
+
+export const test = mergeTests(
+	apiHelpersTest,
+	listTypeDefinitionsPagesTest,
+	loginTest(),
+	siteSettingsPageTests
+);
+
+let createdListTypeDefinitionName: string;
+let customDefaultSiteLanguage: string;
+
+test.afterEach(async ({apiHelpers, page, siteSettingsLocalizationPage}) => {
+	if (customDefaultSiteLanguage) {
+		await page.goto('/');
+
+		await siteSettingsLocalizationPage.goto();
+
+		await siteSettingsLocalizationPage.selectDefaultLanguageOption();
+
+		await siteSettingsLocalizationPage.saveConfiguration();
+	}
+
+	const listTypeDefinitions =
+		await apiHelpers.listTypeAdmin.getListTypeDefinitions();
+
+	const [createdListTypeDefinition] = listTypeDefinitions.items.filter(
+		(listTypeDefinition: PickList) =>
+			listTypeDefinition.name === createdListTypeDefinitionName
+	);
+
+	await apiHelpers.listTypeAdmin.deleteListTypeDefinition(
+		createdListTypeDefinition.id
+	);
```

Comment:
I think that it could be a good thing to put all this block inside an if checking if `createdListTypeDefinitionName` is defined, like you did above with `customDefaultSiteLanguage`:

```
if (createdListTypeDefinitionName) {
    const listTypeDefinitions =
		await apiHelpers.listTypeAdmin.getListTypeDefinitions();

	const [createdListTypeDefinition] = listTypeDefinitions.items.filter(
		(listTypeDefinition: PickList) =>
			listTypeDefinition.name === createdListTypeDefinitionName
	);

	await apiHelpers.listTypeAdmin.deleteListTypeDefinition(
		createdListTypeDefinition.id
	);
}
```